python-sdk/tutorials/nuscenes_tutorial.ipynb

```diff
@@ -35,6 +35,30 @@
     "![](https://www.nuscenes.org/public/images/nuscenes-schema.svg)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Google Colab (optional)\n",
+    "[![Google Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/nutonomy/nuscenes-devkit/blob/master/python-sdk/tutorials/nuscenes_tutorial.ipynb)\n",
```

Comment:
![logo](https://user-images.githubusercontent.com/39502217/93984625-a3129280-fdb6-11ea-8a8d-40b2a360d9e2.png)
The Colab logo looks centered. Can you try to left-align it?

Reply:
Done